src/org/labkey/test/pages/admin/ImportFolderPage.java

```diff
@@ -70,6 +70,8 @@ public void clickImportFolder()
         clickButton("Import Folder");
     }
 
+    public void clickManageFileWatcherTriggers() { clickAndWait(elementCache().manageFileWatcherTriggersButton); }
```

Comment:
When we have a Page class to encapsulate the destination page, this function will ideally return an instance of it